gameroom.js

```diff
@@ -182,6 +184,8 @@ class GameRoom {
         player2_name: this.players[1].name,
         player2_id: this.players[1].id,
         player2_deck_id: this.players[1].deck_id,
+        starting_timer: this.starting_timer,
+        enforce_timer: this.enforce_timer
```

Comment:
Can you add one more timer parameter?
I want a "overtime_per_move" as well.
The idea being that if you run out of time, you still get this amount of time per move.
If you run out of that, then you really lose!
Default should be like 15 or 20 seconds, but you can add it to your settings too.

Reply:
that is a great idea

Reply:
what do you think of phrasing it this way? Minimum time given to take turn: if your turn starts and you're below x seconds, your timer gets set to x seconds

Reply:
I'd like to make it clear in the UI and settings that this is overtime time that occurs after your full timer runs out.

Reply:
Oh, I meant making it actually work in the way my phrasing suggests, apologies. I went ahead and coded minimum time per turn before I saw this - but I'm happy to add in overtime instead, or as an additional option